examples/tf/trpo_swimmer_ray_sampler.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with TRPO algorithm.
+
+Uses Ray sampler instead of on_policy vectorized
+sampler.
+Test
+Here it runs Swimmer-v2 environment with 40 iterations.
+"""
+import gym
+
+from garage.experiment import LocalRunner, run_experiment
+from garage.np.baselines import LinearFeatureBaseline
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import GaussianMLPPolicyWithModel
+from garage.tf.samplers import RaySamplerTF
+
+seed = 100
+
+
+def run_task(*_):
+    """."""
+    with LocalRunner() as runner:
```

Comment:
We need to rebase this to use `LocalTFRunner`

Reply:
Addressed in #830